src/test/java/org/openrewrite/kotlin/.editorconfig

```diff
@@ -3,3 +3,4 @@ root = true
 [*.java]
 indent_size = 4
 ij_continuation_indent_size = 2
+# ij_continuation_indent_size = 8
```

Comment:
I added an option that may be uncommented to apply formatting to TypeMapping and SignatureBuilder tests.

The config only exists to accelerate implementing `RewriteTest` formatting. Standard Java formatting still applies to other Java code.